src/main/java/estaru/birthdaygreetings/Reminder.java

```diff
@@ -0,0 +1,8 @@
+package estaru.birthdaygreetings;
+
+import java.util.List;
+
+public interface Reminder {
```

Comment:
The name of interface do not match with the methods. Reminder should remind, not send birthday message. 

I would organise it like 
- one interface - sender (emailSender/smsSender) with just one method `send`
- second interface/service for creating message to send.